route/account/_id/_login-cookie.js

```diff
@@ -0,0 +1,69 @@
+"use strict";
+
+module.exports = (config) => {
+    var cookieName, cookieSettings;
+
+    /**
+     * Clears the login cookie
+     *
+     * @param {Object} req Request
+     * @param {Object} res Response
+     */
+    function clear(req, res) {
+        if (req.cookies[cookieName]) {
+            res.setCookie(cookieName, "");
+        }
+    }
+
+
+    /**
+     * Returns the login cookie value from the request.
+     *
+     * @param {Object} req Request
+     * @return {string} May be an empty string.
+     */
+    function get(req) {
+        return req.cookies[cookieName];
+    }
+
+
+    /**
+     * Refreshes the login cookie, allowing it to last longer in the browser.
+     *
+     * @param {Object} req Request
+     * @param {Object} res Response
+     * @throws {Error} No cookie value to refresh
```

Comment:
You set `@throws {Error}` here but in other places where a throw happens it is not documented. Would you go through and document other places where we throw?